### DISCUSSION TITLE
Issue working with templates

### DISCUSSION THREAD

**Discussion by utbrott** [JanDeDobbeleer/oh-my-posh#2226] [Troubleshoot]
Hello,
I'm trying to create my own OMP theme using templates, but they seem to not be working correctly. Also for some reason I seem to have problem with palette (but this can really be omitted if I get the templates to work). What I don't udnerstand is why templates seem to be working in parts copied either from docs or someone else's config, but when I try to create mine it fails.

Attached fragment of theme config file in which I'm trying to use templates and the output in Windows Terminal using Powershell 7.2.3.
Using newest version of OMP.

```
{
  "style": "plain",
  "template": "<p:white>\u256d\u2500[</><p:brightBlue>{{ .Name }}</><p:white>] \u2500 </>",
  "type": "shell"
}
```
![image](https://user-images.githubusercontent.com/48184327/167046342-80383f91-c4de-4b2b-ba43-9052233ec285.png)

Expected look:
Off-white (#F3F6FE) lines and brackets with bright blue (#61AFEF) shell name.

Ignore properly formatted parts, they are copied as reference for later. I plan to use plain segments only.

**Comment by JanDeDobbeleer** [comment]
Are you sure you're editing the correct config file?

**Reply by utbrott** [reply]
Yes, I'm working inside my own config file.
It is being loaded as such:
```powershell
# Load prompt config (Oh My Posh)
Import-Module posh-git
function Get-ScriptDirectory { Split-Path $MyInvocation.ScriptName }
$THEME_CONFIG = Join-Path (Get-ScriptDirectory) 'theme.omp.json'
oh-my-posh --init --shell pwsh --config $THEME_CONFIG | Invoke-Expression
```

I was referencing devaslife's video to set the profile loading up.

**Reply by JanDeDobbeleer** [reply]
@utbrott can I get the full config?
